packages/core/content-type-builder/admin/src/pages/ListView/index.js

```diff
@@ -226,16 +225,6 @@ const ListView = () => {
           primaryAction={
             isInDevelopmentMode && (
               <Stack horizontal size={2}>
-                <Button
```

Comment:
Since we have removed the cancel button I think we can also remove all the code associated with the feature as well in the DataManagerProvider component

Reply:
Done @soupette !